### DISCUSSION TITLE
add SoundCloud provider

### DISCUSSION THREAD

**Discussion by stardvst** [nextauthjs/next-auth#3947] [Ideas]
### Description 📓

Please extend music streaming platform authentication support and add SoundCloud OAuth provider: https://developers.soundcloud.com/docs/api/guide#authentication

### How to reproduce ☕️

N/A

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by balazsorban44** [comment]
It's super easy to create a provider config, and if you are satisfied, you can also commit it back here in a PR to make it available to anyone! Check out this page https://next-auth.js.org/configuration/providers/oauth-provider

**Comment by stardvst** [comment]
okay thanks, will take a look

**Comment by stardvst** [comment]
this is tracked in https://github.com/nextauthjs/next-auth/pull/3278

**Comment by stale** [comment]
Hi there! It looks like this issue hasn't had any activity for a while. It will be closed if no further activity occurs. If you think your issue is still relevant, feel free to comment on it to keep it open. (Read more at #912) Thanks!

**Comment by stardvst** [comment]
i will take this soon.

**Comment by stardvst** [comment]
i have added the provider, but currently SoundCloud doesn't allow registering a new app (due to high loading of requests), so i cannot test the implementation for now: https://soundcloud.com/you/apps/new
